src/adhocracy_mercator/adhocracy_mercator/evolution/__init__.py

```diff
@@ -55,14 +55,38 @@ def evolve2_disable_add_proposal_permission(root):  # pragma: no cover
                 (Deny, 'role:creator', 'add_mercator_proposal_version')]
     updated_acl = deny_acl + acl
     set_acl(root, updated_acl, registry=registry)
-    # TODO substanced bug, _p_changed is not set
-    root._p_changed = True
 
     logger.info('Finished substanced evolve step 2:'
                 'remove add_proposal permission')
 
 
+def evolve3_disable_voting_and_commenting(root):
+    """Disable rate and comment permissions."""
```

Comment:
The text suffix evolve.. is not helping when reading the code. Its clear that this is a evolution step
anyway. The function name should express the one thing thate the function is doing.
The number suffix does not have a real meaning and makes merging more difficult

I propose we generelly remove the suffix for migration functions.